src/sentry/analytics/events/alert_created.py

```diff
@@ -1,22 +1,22 @@
 from sentry import analytics
 
 
+@analytics.eventclass("alert.created")
 class AlertCreatedEvent(analytics.Event):
-    type = "alert.created"
-
-    attributes = (
-        analytics.Attribute("user_id", required=False),
-        analytics.Attribute("default_user_id"),
-        analytics.Attribute("organization_id"),
-        analytics.Attribute("rule_id"),
-        analytics.Attribute("rule_type"),
-        analytics.Attribute("is_api_token"),
-        # `alert_rule_ui_component` can be `alert-rule-action`
-        analytics.Attribute("alert_rule_ui_component", required=False),
-        analytics.Attribute("duplicate_rule", required=False),
-        analytics.Attribute("wizard_v3", required=False),
-        analytics.Attribute("query_type", required=False),
-    )
+    user_id: int | None = None
+    default_user_id: int | str | None = None
+    organization_id: int
+    project_id: int
```

Comment:
in this PR we have the `project_id` being `None` too...will this one be always defined?